test/functional/fixtures/reporter/testcafe-fixtures/skipped-tests.js

```diff
@@ -0,0 +1,22 @@
+fixture `Skipped tests`
+    .page `http://localhost:3000/fixtures/reporter/pages/index.html`;
+test('Simple test', async t => {
+    await t.wait(1);
+    await t.report();
+});
+test.skip('Simple command err test', async t => {
```

Comment:
I guess it's too much.
We need covering only two cases: skipped test has duration 0 ms, non-skipped - not 0 ms.